zookeeper-server/src/main/java/org/apache/zookeeper/ClientCnxn.java

```diff
@@ -18,6 +18,7 @@
 
 package org.apache.zookeeper;
 
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
```

Comment:
Do we have to migrate to this annotation-based approach of defining filters?
Can't we just re-use the existing findBugs filter file?

Reply:
@anmolnar personally I like to use annotation because when you read the java file you see the problems.
And I am usually using the configuration file for coarse grained  control, like when there is some kind of check you don't want to consider at all or in many files

Reply:
I'm willing to accept it, but it makes the patch slightly bigger and I feel like we need some additional support from the community in this case. At least one more approval from a committer.